ISound.js

```diff
@@ -181,3 +147,36 @@ ISound.prototype.stop = function (cb) {
 	this.playing = false;
 	return cb && cb();
 };
+
+//▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+/** Load sound. Abstract method to be overwritten
+ * @private
+ */
+ISound.prototype._load = function () {
+	console.log('ISound load call: ' + this.id);
```

Comment:
there is a fallback to this class if neither WebAudio nor HTML audio is available.
I don't think we should Throw in that case.